spec.bs

```diff
@@ -1107,9 +1180,19 @@ To <dfn>parse an https origin</dfn> given a [=string=] |input|:
 To <dfn>build bid generators map</dfn> given an [=auction config=] |auctionConfig|:
 1. Let |bidGenerators| be a new [=ordered map=] whose [=map/keys=] are [=origins=] and whose
   [=map/values=] are [=per buyer bid generators=].
+1. Let |negativeTargetInfo| be a new [=map=] whose [=map/keys=] are [=tuples=] consisting of an

```

Comment:
nit: It might be nice to just add a `Note` documenting exactly why the structure of this is what it is, and how it will be used to achieve the goals related to "negative targeting", just for ease of readability. Or alternatively, linking to a section of the spec that describes what negative targeting is would be nice.